module#2/src/utils/user-operations.ts

```diff
@@ -20,3 +21,13 @@ export const getAutoSuggestUsers = (loginSubstring: string, limit?: number) => {
     return dataBase.users.splice(0, Number(limit));
   }
 };
+
+export const sortingUsersByLoginNameFunction = (a: User, b: User) => {
```

Comment:
Can you simplify this function with usage of built-in JS methods like **sort** and **localCompare**?

Reply:
Make sense, thanks  a lot, very useful remark) Done)